modules/Expectation.js

```diff
@@ -389,8 +389,21 @@ class Expectation {
     )
 
     assert(
-      spy.calls.some(call => isEqual(call.arguments, expectedArgs)),
-      'spy was never called with %s',
+      spy.calls.length > 0,
+      ('spy was not called')
+    )
+    const genMessage = () => {
+      const actualArguments = spy.calls
+        .map(call => `- ${JSON.stringify(call.arguments)}`)
```

Comment:
json doesn't support functions, or null, and a few other JS types - is this the best method to represent the items?

you may want to look into the `object-inspect` module here.